sros2/test/sros2/commands/security/verbs/test_generate_policy.py

```diff
@@ -28,10 +28,10 @@
 @pytest.mark.skip(reason='temporarily deactivated')
 def test_generate_policy_topics():
     with tempfile.TemporaryDirectory() as tmpdir:
-        # Create a test-specific context so that generate_policy can still init
+        # Create a test-specific enclave so that generate_policy can still init
         context = rclpy.Context()
-        rclpy.init(context=context)
-        node = rclpy.create_node('test_generate_policy_topics_node', context=context)
+        rclpy.init(enclave=enclave)
+        node = rclpy.create_node('test_generate_policy_topics_node', enclave=enclave)
```

Comment:
Darn, few more here, sorry man.

Reply:
yeah I saw that too ^^
I changed them all in https://github.com/ros2/sros2/pull/198/commits/e1a3bac1e474fc7ba4bcbd4bf2fd94a388a93c92
So much effort for disabled code :)

Reply:
No kidding. Thank you!

Reply:
I cheated :)
```
F841 local variable 'context' is assigned to but never used
        context = rclpy.Context()
        ^
```